src/KomikTap/KomikTap.ts

```diff
@@ -0,0 +1,128 @@
+/* eslint-disable linebreak-style */
+import {
+  LanguageCode,
+  SourceInfo,
+  TagType,
+  ContentRating,
+} from "paperback-extensions-common";
+
+import { MangaStream, getExportVersion } from "../MangaStream";
+
+import { KomikTapParser } from "./KomikTapParser";
+
+const KOMIKTAP_DOMAIN = "https://194.233.66.232";
```

Comment:
Use valid domain.

Reply:
Unfortunately, i have tried with proper domain.
But it doesn't work when i test...